src/fn/upload-custom-translations.js

```diff
@@ -81,6 +114,76 @@ function parse(filePath, options) {
   });
 }
 
+function checkTranslations(translations, lang, templatePlaceholders) {
+  let mf = null;
+  try {
+    mf = new MessageFormat(lang);
+  } catch (e) {
+    log.warn(`Cannot check '${lang}' translations: ${e.message}`);
+  }
+  const placeholders = extractPlaceholdersFromTranslations(translations);
+  let formatErrorsFound = 0,  placeholderErrorsFound = 0, emptiesFound = 0;
```

Comment:
Nit: Put each variable on its own line, eg

```suggestion
  let formatErrorsFound = 0;
  let placeholderErrorsFound = 0;
  let emptiesFound = 0;
```